### ISSUE TITLE
issue when running db dataset

### ISSUE THREAD

**Issue by ridoughi** [city-super/Scaffold-GS#85]
Hi guys,
Thank you for sharing your code.
I tried to run the code on the Deep Blending playroom scene using the provided script, but I got an issue:

> Setting up [LPIPS] perceptual loss: trunk [vgg], v[0.1], spatial [off]
/home/Myspace/anaconda3/envs/scaffold_gs/lib/python3.7/site-packages/torchvision/models/_utils.py:209: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and will be removed in 0.15, please use 'weights' instead.
  f"The parameter '{pretrained_param}' is deprecated since 0.13 and will be removed in 0.15, "
/home/Myspace/anaconda3/envs/scaffold_gs/lib/python3.7/site-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and will be removed in 0.15. The current behavior is equivalent to passing `weights=VGG16_Weights.IMAGENET1K_V1`. You can also use `weights=VGG16_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
Loading model from: /home/Myspace/anaconda3/envs/scaffold_gs/lib/python3.7/site-packages/lpips/weights/v0.1/vgg.pth
found tf board
2024-10-27 16:52:28,234 - INFO: args: Namespace(add_color_dist=False, add_cov_dist=False, add_opacity_dist=False, appearance_dim=0, appearance_lr_delay_mult=0.01, appearance_lr_final=0.0005, appearance_lr_init=0.05, appearance_lr_max_steps=30000, checkpoint_iterations=[], compute_cov3D_python=False, convert_SHs_python=False, data_device='cuda', debug=False, debug_from=-1, densify_grad_threshold=0.0002, detect_anomaly=False, ds=1, eval=True, feat_dim=32, feature_lr=0.0075, gpu='-1', images='images', ip='127.0.0.1', iterations=30000, lambda_dssim=0.2, lod=0, lowpoly=False, min_opacity=0.005, mlp_color_lr_delay_mult=0.01, mlp_color_lr_final=5e-05, mlp_color_lr_init=0.008, mlp_color_lr_max_steps=30000, mlp_cov_lr_delay_mult=0.01, mlp_cov_lr_final=0.004, mlp_cov_lr_init=0.004, mlp_cov_lr_max_steps=30000, mlp_featurebank_lr_delay_mult=0.01, mlp_featurebank_lr_final=1e-05, mlp_featurebank_lr_init=0.01, mlp_featurebank_lr_max_steps=30000, mlp_opacity_lr_delay_mult=0.01, mlp_opacity_lr_final=2e-05, mlp_opacity_lr_init=0.002, mlp_opacity_lr_max_steps=30000, model_path='outputs/blending/playroom/baseline/2024-10-27_16:52:22', n_offsets=10, offset_lr_delay_mult=0.01, offset_lr_final=0.0001, offset_lr_init=0.01, offset_lr_max_steps=30000, opacity_lr=0.02, percent_dense=0.01, port=15003, position_lr_delay_mult=0.01, position_lr_final=0.0, position_lr_init=0.0, position_lr_max_steps=30000, quiet=False, ratio=1, resolution=-1, rotation_lr=0.002, save_iterations=[30000, 30000], scaling_lr=0.007, sh_degree=3, source_path='data/blending/playroom', start_checkpoint=None, start_stat=500, success_threshold=0.8, test_iterations=[30000], undistorted=False, update_depth=3, update_from=1500, update_hierachy_factor=4, update_init_factor=16, update_interval=100, update_until=15000, use_feat_bank=False, use_wandb=False, voxel_size=0.005, warmup=False, white_background=False)
Backup Finished!
2024-10-27 16:53:48,139 - INFO: Optimizing outputs/blending/playroom/baseline/2024-10-27_16:52:03
Output folder: outputs/blending/playroom/baseline/2024-10-27_16:52:03 [27/10 16:53:48]
Reading camera 225/225 [27/10 16:53:49]
start fetching data from ply file [27/10 16:53:49]
Loading Training Cameras [27/10 16:53:49]
Loading Test Cameras [27/10 16:54:01]
Initial voxel_size: 0.005 [27/10 16:54:02]
Number of points at initialisation :  35155 [27/10 16:54:02]
*** stack smashing detected ***: terminated
Backup Finished!
2024-10-27 16:54:04,368 - INFO: Optimizing outputs/blending/playroom/baseline/2024-10-27_16:52:22
Output folder: outputs/blending/playroom/baseline/2024-10-27_16:52:22 [27/10 16:54:04]
Reading camera 225/225 [27/10 16:54:05]
start fetching data from ply file [27/10 16:54:05]
Loading Training Cameras [27/10 16:54:05]
Loading Test Cameras [27/10 16:54:16]
Initial voxel_size: 0.005 [27/10 16:54:17]
Number of points at initialisation :  35155 [27/10 16:54:17]
*** stack smashing detected ***: terminated
./train.sh: line 35: 998480 Aborted                 (core dumped) python train.py --eval -s data/${data} --lod ${lod} --gpu ${gpu} --voxel_size ${vsize} --update_init_factor ${update_init_factor} --appearance_dim ${appearance_dim} --ratio ${ratio} --iterations ${iterations} --port $port -m outputs/${data}/${logdir}/$time
./train.sh: line 35: 998719 Aborted                 (core dumped) python train.py --eval -s data/${data} --lod ${lod} --gpu ${gpu} --voxel_size ${vsize} --update_init_factor ${update_init_factor} --appearance_dim ${appearance_dim} --ratio ${ratio} --iterations ${iterations} --port $port -m outputs/${data}/${logdir}/$time

Do you have any idea how I can solve it?

**Comment by inspirelt** [comment]
Sorry but I never met this. Could you please verify that whether your data can be used by the official 3DGS?
